src/app/auth/components/LogIn/LogIn.tsx

```diff
@@ -172,7 +172,11 @@ export default function LogIn(): JSX.Element {
         navigationService.history.push(`/activate/${email}`);
       }
 
-      setLoginError([castedError.message]);
+      setLoginError([
+        castedError.message.includes('Wrong login credentials')
```

Comment:
Instead of comparing the error message to display that message, I think that will be bettter try doing it with the http status code. That way even if the backend change the message it will still work if the numerical code is the same.

Reply:
Changed!